finch/brush.py

```diff
@@ -29,6 +29,15 @@ class Brush:
     angle           : float
     size            : int
 
+    def copy( self ) -> "Brush":
+        return Brush(
+            color=self.color,
```

Comment:
Since `Color` is a `tuple` of primitives, we can safely create a direct reference, since the `tuple` is immutable. Note the difference with `Point` below.